src/ROUTES.ts

```diff
@@ -1885,6 +1885,10 @@ const ROUTES = {
         route: 'workspaces/:policyID/rules/billable',
         getRoute: (policyID: string) => `workspaces/${policyID}/rules/billable` as const,
     },
+    RULES_REIMBURSABLE_DEFAULT: {
+        route: 'settings/workspaces/:policyID/rules/reimbursable',
```

Comment:
```suggestion
        route: 'workspaces/:policyID/rules/reimbursable',
```

https://github.com/user-attachments/assets/ab0c35ab-6242-45e5-958d-ae6365d24e02